### DISCUSSION TITLE
Issues connecting to Client API

### DISCUSSION THREAD

**Discussion by iwpdeveloper** [chatwoot/chatwoot#8664] [Chatwoot APIs] [answered]
I am attempting to reposition the Chatwoot console on my webpage, rather than in the chatbot widget that appears at the bottom of the window. 

According to the documentation I would use the Client API to do this. 

(See: https://www.chatwoot.com/docs/contributing-guide/chatwoot-apis - Quote: "Not happy with the native Chatwoot Website widget? Want to create a customer Chat support option right into your mobile app? Client APIs are here to help.")

The problem I am having is that I have been attempting to connect to the Client API, so that I can get this work for the past 2 weeks. No success. I have tried a lot of different options. 

Here are the steps I have followed to get to this point:
(1) I set up an API inbox in my cloud Chatwoot account. 
(2) I installed the code from: https://github.com/chatwoot/client-api-demo 
The code is not working. 
(3) I tried to connect to the API manually via postman. That did not seem to work either. 

What am I doing wrong? 

Here is a page that I have the demo code installed on: 
https://travelersq.com/ai_assistant_2/

And here is the script: 
https://travelersq.com/wp-content/themes/Divi-child/chatwoot_api/chatwoot_api.js?ver=1704738409

**Comment by iwpdeveloper** [comment, marked as answer]
I was able to fix this problem myself by debugging and making changes to the code that was provided in the client demo. I don't know if the code is out of date. Or if it was never tested properly.

**Reply by sojan-official** [reply]
@iwpdeveloper, the code is relatively old and not actively maintained. We would be happy to take PRs if you are interested.
